packages/endomoat/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@lavamoat/endomoat",
+  "version": "2.3.1",
+  "type": "module",
+  "description": "Converts Lavamoat Policies to Endo Policies",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/LavaMoat/LavaMoat.git",
+    "directory": "packages/endomoat"
+  },
+  "homepage": "https://github.com/LavaMoat/LavaMoat/tree/main/packages/endomoat",
+  "author": "Lavamoat Project",
+  "license": "MIT",
+  "private": true,
+  "workspaces": [
+    "test/fixtures/main"
```

Comment:
this is odd. why do we have it?

Reply:
this is a (possibly better) way to prepare the test fixtures; I wanted to see if it'd work.  we could use this elsewhere and eliminate some prep scripts, it looks like.

Reply:
the question to ask, imo, is "why _not_ do this?" and right now I don't have an answer.